cmd/root.go

```diff
@@ -27,9 +27,15 @@ $ snyk-iac-rules test --help
 4. Run the following command to learn how to build the bundle for the Snyk IaC CLI:
 $ snyk-iac-rules build --help
 
-5. Run the following command to learn how to push the bundle to a Container Registry:
+5. Verify your bundle locally using the Snyk IaC CLI:
+$ snyk iac test --rules --help
+
+6. Run the following command to learn how to push the bundle to an OCI Registry:
 $ snyk-iac-rules push --help
 
+7. Use the bundle stored in the OCI registry in the Snyk IaC CLI:
+$ snyk --help
```

Comment:
is the general snyk cli help command is the right one here?
`snyk --help`

The help commands above looked more specific.

Reply:
If you run `snyk --help` you will get instructions on how to run this with custom rules. Here's the output:
```
   Infrastructure as Code (IAC) scanning
       See snyk iac --help for more details and examples:


           $ snyk iac test /path/to/cloudformation_file.yaml
           $ snyk iac test /path/to/kubernetes_file.yaml
           $ snyk iac test /path/to/terraform_file.tf
           $ snyk iac test /path/to/tf-plan.json
           $ snyk iac test /path/to/arm_file.json



       To use your own custom rules to scan IaC configuration files,  download
       the  snyk-iac-rules  SDK  from  https://github.com/snyk/snyk-iac-rules.
       Follow the instructions there to write, build, and push a custom  rules
       bundle  and  then either use the Snyk UI to configure your custom rules
       settings or configure a remote OCI registry locally by running the fol-
       lowing commands:


           $ snyk config set oci-registry-url=https://registry-1.docker.io/username/repo:tag
           $ snyk config set oci-registry-username=username
           $ snyk config set oci-registry-password=password
```